.gitignore

```diff
@@ -2,3 +2,6 @@
 /.lintcache
 /node_modules
 npm-debug.log
+
+# Ignore sublime project files.
+*.sublime-*
```

Comment:
I agree with @medikoo.

There's also a special per-repository place where you can store ignores, but they would not be under version control:
`.git/info/exclude`
That is if you want to store sublime files in repo, you can put this ignore rule there.